src/braket/circuits/ascii_circuit_diagram.py

```diff
@@ -84,6 +84,10 @@ def build_diagram(circuit) -> str:
         if additional_result_types:
             lines.append(f"\nAdditional result types: {', '.join(additional_result_types)}")
 
+        # A list of parameters in the circuit to the currently assigned values.
+        for parameter in circuit.parameters:
+            lines.append(f"\n{parameter} has no assigned value.")
```

Comment:
Probably want something like this; otherwise you might end up with many more lines than is readable.
```suggestion
        lines.append(f"Unassigned parameters: {circuit.parameters}")
```